File: API/Models/Location.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace API.Models
{
    public class Location
    {
        public string type { get; set; } = "Point";
        public double[] coordinates { get; set; } = { 0, 0 };
     
        public Location(double longitude, double latitude)
        {
            coordinates[0] = longitude;
            coordinates[1] = latitude;
        }

        public double getLongitude()
        {
            return coordinates[0];
        }

        public double getLatitude()
        {
            return coordinates[1];
        }

    }
}

File: Clients/Vendelo_RD_android/app/src/main/java/com/practices/samuel/vendelo_rd_android/Model/Location.kt
package com.practices.samuel.vendelo_rd_android.Model

/**
 * Created by Samuel on 12/23/2016.
 */
class Location(longitude:Double,latitude:Double) {
    val type:String="Point"
    val coordinates:DoubleArray = doubleArrayOf(0.0, 0.0)

    init{
        coordinates[0]=longitude
        coordinates[1]=latitude
    }

    fun getLongitude(): Double {
        return coordinates[0]
    }

    fun getLatitude(): Double {
        return coordinates[1]
    }
}